awx/main/dispatch/pool.py

```diff
@@ -373,6 +373,10 @@ def cleanup(self):
             # don't use our logger (it accesses the database for configuration)
             _, _, tb = sys.exc_info()
             traceback.print_tb(tb)
+            for conn in connections.all():
```

Comment:
Without this, a database blip can cause the connection pool to not reset here, and can cause a job to get stuck in `running`.  In real world scenarios, this corrects itself by launching _other_ jobs (because there's code _elsewhere_ that detects these stale connections and resets them).